src/garage/experiment/experiment.py

```diff
@@ -339,20 +340,26 @@ def my_experiment(ctxt, seed, lr=0.5):
         archive_launch_repo (bool): Whether to save an archive of the
             repository containing the launcher script. This is a potentially
             expensive operation which is useful for ensuring reproducibility.
+        name_parameters (str or None): Parameters to insert into the experiment
+            name. Should be either None (the default), 'all' (all parameters
+            will be used), or 'passed' (only passed parameters will be used).
+            The used parameters will be sorted, and inserted with their values
+            into the experiment name.
```

Comment:
Main documentation of the new parameter is here.

Reply:
And actually is slightly wrong (the parameters aren't sorted, they're used in the order of the function definition). I'll fix the doc in a minute.